Portfolio: setup and mobile version skeleton

samlorlah opened the pull request:
**Linters, .gitignore and ReadMe**

- Html-CSS Linters was added and commit with a branch and comment

- .gitignore was added with node_modules as folder to ignore

**HTML**
Html Page was added and 2 sections were added. The header for the logo and Nav Hamburger and the first section of the website

**Stylesheet**
Stylesheet file was added in asset/css and the 2 sections were styled according to the template figma.

samlorlah committed:
Add HTML-CSS Linters to the Project

samlorlah committed:
git ignore added to ignore unwanted files like node modules

samlorlah committed:
Index Html File Added

Asset Folder with Image folder added

samlorlah committed:
Stylesheet added to for the header and first section - Mobile View

samlorlah committed:
Stylesheet added to for the header and first section - Mobile View

samlorlah committed:
Stylesheet added to for the header and first section - Mobile View

samlorlah committed:
Stylesheet added to for the header and first section - Mobile View

vechicin commented on index.html:
```diff
@@ -0,0 +1,46 @@
+<body>
+    <header>
+        <div>
+            <span id="logo">My Logo</span>
```
This content should be wrapped in an `<a>` tag. Please fix it.

<img width="442" alt="Screen Shot 2022-06-29 at 18 15 23" src="https://user-images.githubusercontent.com/82426682/176561197-56ab4e62-3ada-4f99-88ef-6340dd6eb499.png">

samlorlah replied:
Done

vechicin commented on asset/css/style.css:
```diff
@@ -0,0 +1,99 @@
+    padding-right: 5px;
+  }
+
+  #main-section {
+    background-image: url(../images/background.png);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 100% 95%;
+    height: 588px;
+  }
```
Your background image placement is not quite right. Please take a closer look at the Figma design and fix it.

<img width="304" alt="Screen Shot 2022-06-29 at 18 18 26" src="https://user-images.githubusercontent.com/82426682/176561524-b16e6bc6-ed4c-43c5-ad95-004cfafcfdc1.png">

Additionally, you're missing the `border-radius` in the lower-left corner of the section, please add it.

samlorlah replied:
Fixed

vechicin commented on index.html:
```diff
@@ -0,0 +1,46 @@
+            <div id="connect">
+                <h3>Let's Connect</h3>
+            </div>
+            <div class="social-media-icons">
+                <a href="#"><img src="./asset/images/Vectortwitter.svg" alt="twitter"></a>
+                <a href="#"><img src="./asset/images/linkedin.svg" alt="linkedin"></a>
+                <a href="#"><img src="./asset/images/Vectorm-icon.svg" alt="M"></a>
+                <a href="#"><img src="./asset/images/Vectorgithub.svg" alt="github"></a>
+                <a href="#"><img src="./asset/images/Vectordribble.svg" alt="dribble"></a>
+            </div>
```
This content should be wrapped in several `<li>` all of them wrapped in a `<ul>`. Please fix it.

<img width="603" alt="Screen Shot 2022-06-29 at 18 19 59" src="https://user-images.githubusercontent.com/82426682/176561696-d5661b70-f97e-41c3-9d7c-f4c46c0f4bb8.png">

samlorlah replied:
Done

vechicin commented on asset/css/style.css:
```diff
@@ -0,0 +1,99 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+/* Mobile View */
+@media all and (max-width: 375px) {
+  * {
+    padding-left: 5px;
+    padding-right: 5px;
+  }
```
Your design has margins that are not present in the template design, please remove them.

<img width="486" alt="Screen Shot 2022-06-29 at 18 23 00" src="https://user-images.githubusercontent.com/82426682/176561985-7d946973-5648-445c-b145-befd29224efa.png">

samlorlah replied:
Removed

vechicin reviewed (changes requested):
# Hi @samlorlah ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Clean codebase
- [x] Good job overall following the template design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

samlorlah committed:
Corrections from the first pull request done

Putting Logo in a tag

Main Section Background Resolved

Social Media Icon Moved to ul>li tag

Unwanted Margin Removed

samlorlah committed:
Corrections from the first pull request done

Putting Logo in a tag

Main Section Background Resolved

Social Media Icon Moved to ul>li tag

Unwanted Margin Removed

sumairq reviewed (approved):
### Hi @samlorlah  , 

<img src="https://media2.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
